src/megameklab/com/ui/Mek/tabs/StructureTab.java

```diff
@@ -659,12 +659,12 @@ public void updateTechLevel() {
         }
         panChassis.refresh();
         panHeat.refresh();
-        panMovement.refresh();
         panArmor.refresh();
         panArmorAllocation.setFromEntity(getMech());
         panPatchwork.setFromEntity(getMech());
         refresh.refreshBuild();
         addAllListeners();
+        panMovement.refresh();
```

Comment:
Does this cause any issues being below refreshBuild and addAllListeners?

Reply:
The issue was caused in part by it being above that.